nixos/modules/services/misc/paperless.nix

```
@@ -571,6 +584,10 @@ in
             gid = config.ids.gids.paperless;
           };
         };
+
+        services.gotenberg.enable = lib.mkIf cfg.enableExtraFilesOCR true;
+        services.tika.enable = lib.mkIf cfg.enableExtraFilesOCR true;
+        services.tika.enableOcr = lib.mkIf cfg.enableExtraFilesOCR true;
```

```suggestion
        services.gotenberg = lib.mkIf cfg.enableExtraFilesOCR {
          enable = true;
        };

        services.tika = lib.mkIf cfg.enableExtraFilesOCR {
          enable = true;
          enableOcr = true;
        };
```
In case we configure more options later one.